### DISCUSSION TITLE
I cannot uninstall

### DISCUSSION THREAD

**Discussion by vporton** [wixtoolset/issues#7308] [Questions]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> 4.0.0-rc.4+dea44a7d

* Which version of Visual Studio are you building with (if any)?

> none

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> none

* Which version of .NET are you building with?

> none

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> 10

* Describe the problem and the steps to reproduce it.

The software I created installs successfully, but cannot be uninstalled. The uninstaller rollbacks.

It seems you have an incompatibility with NSSM. (I'll report to them, too.)

1. Download https://github.com/vporton/bug-msi-uninst
2. Compile it with `make` (you need GNU Make and GCC).
3. Run installer (it does not ask any questions).
4. Run uninstaller.

* Describe the behavior you expected and how it differed from the actual behavior.

Successful installing and uninstalling.

**Comment by BMurri** [comment]
Need verbose logs of both the install and the uninstall.
